### ISSUE TITLE
Loader of pretrain Resnet on imageNet

### ISSUE THREAD

**Issue by thegodone** [ml-explore/mlx-examples#132]
Would be nice to have a loader of pretrain weights for Resnet.

**Comment by bigsnarfdude** [comment]
@thegodone found some code for alexnet and vgg with loading of pretrained here. might provide hints on a loader https://github.com/robertmccraith/mimm/tree/main/mimm/models

**Comment by awni** [comment]
This would be nice to have as an example if anyone wants to contribute it. Once we have Batch Normalization in `mlx.nn` which is a PR right now, it should be pretty straightforward.
